sdk/communication/azure-communication-identity/samples/identity_samples.py

```diff
@@ -14,21 +14,33 @@
 USAGE:
     python identity_samples.py
     Set the environment variables with your own values before running the sample:
-    1) COMMUNICATION_SAMPLES_CONNECTION_STRING - the connection string in your ACS resource
+    1) COMMUNICATION_SAMPLES_CONNECTION_STRING - the connection string in your Communication Services resource
     2) AZURE_CLIENT_ID - the client ID of your active directory application
     3) AZURE_CLIENT_SECRET - the secret of your active directory application
     4) AZURE_TENANT_ID - the tenant ID of your active directory application
+    5) COMMUNICATION_M365_APP_ID - the application id of M365
+    6) COMMUNICATION_M365_AAD_AUTHORITY - the AAD authority of M365  
+    7) COMMUNICATION_M365_AAD_TENANT - the tenant ID of M365 application
+    8) COMMUNICATION_M365_SCOPE - the scope of M365 application
+    9) COMMUNICATION_MSAL_USERNAME - the username for authenticating via MSAL library
+    10)COMMUNICATION_MSAL_PASSWORD - the password for authenticating via MSAL library 
```

Comment:
```suggestion
    10) COMMUNICATION_MSAL_PASSWORD - the password for authenticating via MSAL library 
```

Reply:
or indent everything by one more space if you want to have it vertically aligned :D